src/game.cpp

```diff
@@ -2689,15 +2689,20 @@ void Game::handleAndroidChatInput()
 
 void Game::toggleFreeMove()
 {
-	static const wchar_t *msg[] = { L"free_move disabled", L"free_move enabled" };
-
 	bool free_move = !g_settings->getBool("free_move");
 	g_settings->set("free_move", bool_to_cstr(free_move));
 
 	runData.statustext_time = 0;
-	m_statustext = msg[free_move];
-	if (free_move && !client->checkPrivilege("fly"))
-		m_statustext += L" (note: no 'fly' privilege)";
+
+	if (free_move)
+	{
```

Comment:
Please follow the core/Linux style guidelines: http://dev.minetest.net/Code_style_guidelines
```C++
if (condition) {
	foo
} else {
	bar
}
```